cypress/integration/Header.spec.js

```diff
@@ -0,0 +1,110 @@
+const viewports = [
+  // Desktop
+  {
+    size: [1440, 900],
+  },
+  // Large mobile phone
+  {
+    preset: 'samsung-s10',
+    orientation: 'portrait',
+  },
+  {
+    preset: 'samsung-s10',
+    orientation: 'landscape',
+  },
+  // Small mobile phone
+  {
+    preset: 'iphone-3',
+    orientation: 'portrait',
+  },
+  {
+    preset: 'iphone-3',
+    orientation: 'landscape',
+  },
+  // Tablet
+  {
+    preset: 'ipad-2',
+    orientation: 'portrait',
+  },
+  {
+    preset: 'ipad-2',
+    orientation: 'landscape',
+  },
+];
+
+const setViewPort = (viewport) => {
+  if (viewport.size) {
+    cy.viewport(viewport.size[0], viewport.size[1]);
+  } else {
+    cy.viewport(viewport.preset, viewport.orientation);
+  }
+};
+
+const getViewPortName = (viewport) => {
+  if (viewport.size) {
+    return viewport.size;
+  }
+  return `${viewport.preset}, ${viewport.orientation}`;
+};
+
+describe('Header components', () => {
+  beforeEach(() => {
+    cy.visit('');
+  });
+
+  // For each viewport
+  viewports.forEach((viewport) => {
+    describe(`${getViewPortName(viewport)} screen`, () => {
+      it(`should display menu`, () => {
+        setViewPort(viewport);
+        cy.get('.wkp-menu-header').should('be.visible');
+        cy.get('.wkp-topics-menu-body').should('not.be.visible');
+        cy.get('.wkp-menu-header').click();
+
+        // Menu should open
+        cy.get('.wkp-topics-menu-body').should('be.visible');
+      });
+
+      it(`should display WKP login`, () => {
+        setViewPort(viewport);
+        cy.get('.wkp-login').should('be.visible');
```

Comment:
We should either find a way of testing the redirect or simply not write the test at all I think.

Reply:
I have checked this. These [code samples](https://github.com/geops/trafimage-maps/blob/master/src/components/Login/Login.js#L19) show solutions how to test redirects. As the Login component does not have a href attribute and opens the new page by using window.location.href (see [line](https://github.com/geops/trafimage-maps/blob/master/src/components/Login/Login.js#L19)), we could only use [third option](https://github.com/cypress-io/cypress-example-recipes/blob/master/examples/testing-dom__tab-handling-links/cypress/integration/tab_handling_anchor_links_spec.js#L183) which sets chromeWebSecurity to false. However, when I do this, I get redirected to chrome-error://chromewebdata/. I could try to solve this. But, I think, the main idea of cypress is to test a single domain. 

I would prefer to only check if this Login button is available. It's better than checking nothing.